cmake/projects/SDL2/hunter.cmake

```diff
@@ -193,6 +193,50 @@ hunter_add_version(
         04d7768f4418ba03537ef14a86a0c1c45582f5c3
 )
 
+hunter_add_version(
+        PACKAGE_NAME
+        SDL2
+        VERSION
+        "2.24.2"
+        URL
+        "https://github.com/libsdl-org/SDL/releases/download/release-2.24.2/SDL2-2.24.2.tar.gz"
+        SHA1
+        828974badaf1f16f8e3ebe84fb6f7e5d793c1e47
+)
+
+hunter_add_version(
+        PACKAGE_NAME
+        SDL2
+        VERSION
+        "2.26.5"
+        URL
+        "https://github.com/libsdl-org/SDL/releases/download/release-2.26.5/SDL2-2.26.5.tar.gz"
+        SHA1
+        ca5d89edc537fd819eddab1f1a86f61e45fcb68b
+)
+
+hunter_add_version(
+        PACKAGE_NAME
+        SDL2
+        VERSION
+        "2.28.5"
```

Comment:
No need to add all the versions. Just the newest is enough

Reply:
Those are major versions only. They break some stuff from version to version, and it's easier to try different versions through config than have a git submodule. I have checked building with all intermediate versions.

Reply:
ah ok, then I'm merging. Thanks for the clarification